### ISSUE TITLE
Need to Log In to Stop Workspace

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-che/che#13971]
### Describe the bug
Sometimes che asks me to log in again when I want to expand the workspace management sidebar ("show navigation bar") in the top left corner.  Even when I've been working in my workspace a short time before.
### Che version
 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Expected behavior
I would not expect to have to log in at this point

### Runtime
   - [ ] kubernetes (include output of `kubectl version`) 
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl 
   - [x] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

**Comment by nickboldt** [comment]
Are you talking about showing the Dashboard?

**Comment by skabashnyuk** [comment]
I think this issue is similar to https://github.com/eclipse/che/issues/12099.

**Comment by tsmaeder** [comment]
Yes, the problem really is that working in the IDE does not seem to count as working in the login session. The linked issue was closed because there is a workaround, as I understand. Still a problem, then.

**Comment by tsmaeder** [comment]
lifecycle/frozen
